spec/services/issuing_ca_service_spec.rb

```diff
@@ -95,6 +95,17 @@
     end
   end
 
+  describe '.certificate_store_issuers' do
+    it 'allowed hosts configuration contains all certificate store issuers' do
```

Comment:
These won't be equal, but the config should probably at least include the issuers from the certs we have stored?

Wasn't sure of a better way to handle this, or if it was worth testing in this way.

Reply:
Maybe `.reset` the certificate store to clear the certs before loading them so only the ones we want are there?